homeassistant/components/rmvtransport/sensor.py

```diff
@@ -135,7 +129,12 @@ def __init__(
             max_journeys,
             timeout,
         )
+        self._show_on_map = show_on_map
         self._icon = ICONS[None]
+        self._uniqueid = slugify(
+            f"s{station}-d{direction}-l{''.join([str(l) for l in lines])}-"
+            f"p{''.join(products)}-t{time_offset}-mj{max_journeys}-{name}"
```

Comment:
If name is user configurable without connection to the resource features, eg station etc, we shouldn't include it in the unique id. The unique id must both be unique and stable.